src/common/components/cards/fix-instruction-color-box.scss

```
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../common/styles/colors.scss';
```

I think the `../../..` can be omitted:
```suggestion
@import 'common/styles/colors.scss';
```

---

that works for typescript file... we haven't done anything like this for scss... don't get me wrong, it may work, but also, it may not